manifest.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
I prefer to use Secret instead of a ConfigMap, we have some secrets here, like the SECRET_KEY or the DB PASSWORD.

Reply:
Fine for me :)

Reply:
Done!